### PR TITLE
Extra communication phrases and some skill descriptions

### PR DESCRIPTION
**Pull request by phuclagger**
Translate the new 2 communication phrase pallets. Some phrases have two versions that are slightly different (for male and female). 

Skills:
- Attempted to reuse the 'Need Review' skill description for [Demon Arrow, Shrukis Guard, Alchemical Burst, Annihilator's Wind Slash].
- Translate the 'Effective against War-ready enemies' line of [Downthurst, Demon Arrow, Shrukis Guard, Alchemical Burst].
- Fix the mistranslation of 'Superior Evasion' description.

### CODE REVIEW

File: ui/00_message/simple_com/chat_com_text_07.xml [hunk 1]
```diff
@@ -2,265 +2,331 @@
...
   </message>
   <message>
     <key>CHAT_COM_277</key>
     <original>いち、にの、さんはい！</original>
+    <translation>One more time!</translation>
   </message>
   <message>
     <key>CHAT_COM_278</key>
     <original>みんな集まれ！　こっちだ！</original>
+    <translation>Everyone gather overhere!</translation>
```
Comment (riftcrystal) [inline ui/00_message/simple_com/chat_com_text_07.xml]: Missing a space.

File: ui/00_message/simple_com/chat_com_text_07.xml [hunk 2]
```diff
@@ -2,265 +2,331 @@
...
   </message>
   <message>
     <key>CHAT_COM_279</key>
     <original>みんな集まって、こっちこっち！</original>
+    <translation>Everyone gather overhere!</translation>
```
Comment (riftcrystal) [inline ui/00_message/simple_com/chat_com_text_07.xml]: Missing a space.

File: ui/00_message/skill/custom_skill_command_08.xml [hunk 3]
```diff
@@ -86,9 +86,13 @@ and then unleash with &lt;KC SHOOT&gt;</translation>
   <message>
     <key>CUSTOM_SKILL_COMMAND_08_184</key>
     <original>セットボタンを押して詠唱後、&lt;KC SHOOT&gt;で放つ</original>
+    <translation>Press the set button to begin casting,
```
Comment (riftcrystal) [inline ui/00_message/skill/custom_skill_command_08.xml]: Why the change from "assigned" to "set"? (I won't repeat this message for the others)

File: ui/00_message/skill/custom_skill_info_01.xml [hunk 4]
```diff
@@ -69,7 +69,7 @@ When an attack is evaded, increases number of hits.&lt;/COL&gt;</translation>
 &lt;COL ffdc78&gt;LV UP毎に攻撃威力の上昇
 LV.６で攻撃ヒット数増加&lt;/COL&gt;</original>
     <translation>Leap forward and stab your sword into the ground.
-&lt;COL ff7878&gt;戦甲へのカウンターに特化&lt;/COL&gt;
+&lt;COL ff7878&gt;Effective against War-ready enemies.&lt;/COL&gt;
```
Comment (riftcrystal) [inline ui/00_message/skill/custom_skill_info_01.xml]: Ready should be capitalized here.

File: ui/00_message/skill/custom_skill_info_07.xml [hunk 5]
```diff
@@ -144,7 +144,7 @@ LV.6: Increases number of hits.&lt;/COL&gt;</translation>
 LV.６で剣に風圧が追加／攻撃ヒット数増加&lt;/COL&gt;</original>
     <translation>After channeling enough energy, release
 a devastating slash with your body's entire might.
-&lt;COL ff7878&gt;戦甲へのカウンターに特化&lt;/COL&gt;
+&lt;COL ff7878&gt;Effective against War-ready enemies.&lt;/COL&gt;
```
Comment (riftcrystal) [inline ui/00_message/skill/custom_skill_info_07.xml]: Ready should be capitalized here.

**Review by riftcrystal — CHANGES_REQUESTED** [review]

File: ui/00_message/skill/custom_skill_command_08.xml (hunk 3 above)
Comment (phuclagger) [reply ui/00_message/skill/custom_skill_command_08.xml]: I just copy paste the same line from previous command in the same file. Should it be 'assigned' across all classes?

Comment (riftcrystal) [reply ui/00_message/skill/custom_skill_command_08.xml]: I'll say yes, simply because "assigned" sounds much better than "set".

Comment (phuclagger) [reply ui/00_message/skill/custom_skill_command_08.xml]: Alright I'll change them.

**Comment by phuclagger** [comment]
Alright I think I changed all of the stuff you mentioned.
